### ISSUE TITLE
not working on .svelte files

### ISSUE THREAD

**Issue by alexanderniebuhr** [tailwindlabs/tailwindcss-intellisense#252]
I still cannot get it working, sometimes it works, sometimes it doesn't. constantly reloading my vscode is very annoying, do we have anything new here. I get that some other suggestions might interfere, however it should handle this fine and just show suggestions of both extenstions

**Comment by dgellow** [comment]
@alexanderniebuhr Did you already check the various solutions suggested in this discussion? https://github.com/tailwindlabs/tailwindcss-intellisense/issues/151

**Comment by alexanderniebuhr** [comment]
yes, i did check them, it works sometimes, but sometimes it does not. so, time for time I must reload window

**Comment by bradlc** [comment]
Hey @alexanderniebuhr. Are you seeing any errors in the `Output` panel when the extension stops working?

![image](https://user-images.githubusercontent.com/2615508/106312440-8acbec00-625e-11eb-8f95-f026f8ddcd53.png)

**Comment by alexanderniebuhr** [comment]
Haven't seen any errors so far. I will investigate further.

**Comment by losnappas** [comment]
I noticed that my auto completions broke when I didn't have `tailwind.config.cjs`, even though I had `tailwind.config.mjs`.
